mithril.js

```diff
@@ -1706,16 +1706,16 @@
 		var currentTarget = e.currentTarget || e.srcElement
 		var args
 
+		while (currentTarget && !/^a$/i.test(currentTarget.nodeName)) {
```

Comment:
`<area>`does not have content.

`<a>` does have content, but can be used as named anchor and not as a link.

What we are looking for is the closest *link*, so it must have an href attribute.

I know it's not silver proof either, but at least it covers the case of named anchor.